src/app/event/new-scheduled-event/new-scheduled-event.component.ts

```diff
@@ -175,7 +175,7 @@ export class NewScheduledEventComponent implements OnInit {
         continue;
       }
       var initVal = this.se.required_vms[ea.environment]?.[template] ?? 0; // so we don't blow away old input values when rebuilding this form
-      var newControl = new FormControl(initVal, [Validators.pattern(/-?\d+/), Validators.max(ea.available_count[template])]);
+      var newControl = new FormControl(initVal, [Validators.pattern(/-?\d+/), Validators.max(ea.available_count[template]), Validators.min(1)]);
```

Comment:
Couldn't we just update the regex and remove the "-?" in there?

```suggestion
      var newControl = new FormControl(initVal, [Validators.pattern(/\d+/), Validators.max(ea.available_count[template]), Validators.min(1)]);
```